content/en/docs/admin/authorization/rbac.md

```diff
@@ -541,6 +541,11 @@ It does not allow viewing secrets, since those are escalating.</td>
 The permissions required by individual control loops are contained in the <a href="#controller-roles">controller roles</a>.</td>
 </tr>
 <tr>
+<td><b>system:kubelet-api-admin</b></td>
```

Comment:
text looks fine, but I would put this in the "Other Component Roles" section

Reply:
👍